common/src/node.rs

```diff
@@ -77,6 +96,39 @@ impl Graph {
         Some((chunk, coords))
     }
 
+    /// Populates a chunk with the given voxel data and ensures that margins are correctly removed
+    /// if necessary.
+    pub fn populate_chunk(&mut self, chunk: ChunkId, mut new_data: VoxelData, modified: bool) {
+        // New chunks generated by world generation should not be solid if they are adjacent to modified chunks.
```

Comment:
Why not?

Reply:
This has the same reason as https://github.com/Ralith/hypermine/pull/317/files/410344b5bc303361e0854a05b78ea85f4e3ad1af..fe227b1c1149edb2c6aff48248a54d4683e4a736#r1421834861.

It might make the most sense to explain this in more detail in a comment at the top of the method implementation, as multiple separate parts of this method exist for the same reason.